ios/BT/ExposureManager.swift

```diff
@@ -335,25 +335,33 @@ final class ExposureManager: NSObject {
                             completionHandler: completionHandler)
                 return
               }
-              let userExplanation = NSLocalizedString(String.newExposureNotificationBody, comment: .default)
-              self.manager.getExposureInfo(summary: summary!,
-                                               userExplanation: userExplanation) { exposures, error in
-                if let error = error {
-                  self.finish(.failure(error),
+              if let summary = summary, ExposureManager.score(summary: summary, with: configuration) {
+                let userExplanation = NSLocalizedString(String.newExposureNotificationBody, comment: .default)
+                self.manager.getExposureInfo(summary: summary,
```

Comment:
See question to Matt in Slack - I don't know whether we should request getExposureInfo here or not.
We won't be using the result I believe, but this does automatically trigger a notification. Maybe this is how we want to create the notification, maybe it isn't. I think this is fine for now though - it's doing the same thing the existing code does - and if it should be different then that can be done in a follow-up change.